### PR TITLE
From PhantomJS to Chrome!

### PR DESCRIPTION
**Pull request by rogierslag**
Quick benchmark on Magnet.me yielded the following:

1. Request on Chrome take half the time of that on Phantom
1. Memory usage on Chrome is 6x lower than Phantom (I was surprised as well)
1. Chrome can handle two threads without problems
1. Chrome should properly cache static assets instead of re-retrieving them continuously
1. Lower exception rate on Chrome

Also Chrome can use ES6 code, which makes life a bit easier.

First I'm building this branch as a separate container and will deploy canary to production to test this

### CODE REVIEW

**Comment by rogierslag** [comment]
And I'm quite sure I can trim down the image some more

**Comment by rogierslag** [comment]
Benchmark results (only indicative)

Chrome
```
Rogiers-MacBook-Pro:slow-server rogier$ ab -n 50 -c 2 http://localhost:3000/https://magnet.me/a/company/blendle/timeline
This is ApacheBench, Version 2.3 <$Revision: 1796539 $>
Copyright 1996 Adam Twiss, Zeus Technology Ltd, http://www.zeustech.net/
Licensed to The Apache Software Foundation, http://www.apache.org/

Benchmarking localhost (be patient).....done


Server Software:        nginx/1.12.2
Server Hostname:        localhost
Server Port:            3000

Document Path:          /https://magnet.me/a/company/blendle/timeline
Document Length:        175009 bytes

Concurrency Level:      2
Time taken for tests:   76.539 seconds
Complete requests:      50
Failed requests:        49
   (Connect: 0, Receive: 0, Length: 49, Exceptions: 0)
Total transferred:      8611259 bytes
HTML transferred:       8586486 bytes
Requests per second:    0.65 [#/sec] (mean)
Time per request:       3061.542 [ms] (mean)
Time per request:       1530.771 [ms] (mean, across all concurrent requests)
Transfer rate:          109.87 [Kbytes/sec] received

Connection Times (ms)
              min  mean[+/-sd] median   max
Connect:        0    0   0.1      0       0
Processing:  2248 3017 1153.6   2796   10074
Waiting:     2224 3012 1153.3   2795   10073
Total:       2249 3017 1153.6   2796   10074

Percentage of the requests served within a certain time (ms)
  50%   2796
  66%   2842
  75%   2919
  80%   2931
  90%   3121
  95%   5214
  98%  10074
  99%  10074
 100%  10074 (longest request)
```

PhantomJs
```
Rogiers-MacBook-Pro:slow-server rogier$ ab -n 50 -c 2 http://localhost:3000/https://magnet.me/a/company/blendle/timeline
This is ApacheBench, Version 2.3 <$Revision: 1796539 $>
Copyright 1996 Adam Twiss, Zeus Technology Ltd, http://www.zeustech.net/
Licensed to The Apache Software Foundation, http://www.apache.org/

Benchmarking localhost (be patient).....done


Server Software:        nginx/1.12.2
Server Hostname:        localhost
Server Port:            3000

Document Path:          /https://magnet.me/a/company/blendle/timeline
Document Length:        398473 bytes

Concurrency Level:      2
Time taken for tests:   147.500 seconds
Complete requests:      50
Failed requests:        49
   (Connect: 0, Receive: 0, Length: 49, Exceptions: 0)
Non-2xx responses:      2
Total transferred:      20324519 bytes
HTML transferred:       20304009 bytes
Requests per second:    0.34 [#/sec] (mean)
Time per request:       5900.000 [ms] (mean)
Time per request:       2950.000 [ms] (mean, across all concurrent requests)
Transfer rate:          134.56 [Kbytes/sec] received

Connection Times (ms)
              min  mean[+/-sd] median   max
Connect:        0    0   0.0      0       0
Processing:  3232 5852 2962.4   4485   16117
Waiting:     3229 5850 2962.4   4482   16114
Total:       3232 5853 2962.4   4485   16117

Percentage of the requests served within a certain time (ms)
  50%   4485
  66%   6103
  75%   7159
  80%   8010
  90%  10165
  95%  12320
  98%  16117
  99%  16117
 100%  16117 (longest request)
```

File: server.js [hunk 1]
```diff
@@ -3,42 +3,27 @@ const prerender = require('prerender');
...
+server.use(removePrefetchTags);
 server.use(prerender.httpHeaders());
 if (process.env.DEBUG_PAGES) {
 	server.use(prerender.logger());
 }
 server.use(stripHtml);
 
 server.start();
-
-const shutdown = () => {
```
Comment (AlexKolpa) [inline server.js]: No shutdown required?

File: Dockerfile [hunk 2]
```diff
@@ -16,8 +19,9 @@ RUN chown prerender:prerender /usr/src/app
 USER prerender
 WORKDIR /usr/src/app
 
+COPY yarn.lock /usr/src/app/
 COPY package.json /usr/src/app/
-RUN npm install
+RUN yarn install
 COPY . /usr/src/app
 
 CMD [ "npm", "start" ]
```
Comment (AlexKolpa) [inline Dockerfile]: `yarn start`?

File: server.js (hunk 1 above)
Comment (rogierslag) [reply server.js]: chrome does not hang and crash as phantom used to do

**Pull request merged**